src/sentry/static/sentry/app/views/organizationGroupDetails/groupMerged/mergedToolbar.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import pick from 'lodash/pick';
+
+import {openDiffModal} from 'app/actionCreators/modal';
+import Button from 'app/components/button';
+import LinkWithConfirmation from 'app/components/links/linkWithConfirmation';
+import SpreadLayout from 'app/components/spreadLayout';
+import Toolbar from 'app/components/toolbar';
+import {t} from 'app/locale';
+import GroupingStore from 'app/stores/groupingStore';
+import space from 'app/styles/space';
+import {Group, Organization, Project} from 'app/types';
+import {callIfFunction} from 'app/utils/callIfFunction';
+
+type Props = {
+  orgId: Organization['slug'];
+  project: Project;
+  groupId: Group['id'];
+  onUnmerge: () => void;
+  onToggleCollapse: () => void;
+};
+
+type State = {
+  unmergeList: Map<any, any>;
+  unmergeLastCollapsed: boolean;
+  unmergeDisabled: boolean;
+  enableFingerprintCompare: boolean;
+};
+
+class MergedToolbar extends React.Component<Props, State> {
+  state: State = this.getInitialState();
+
+  getInitialState() {
+    const {
+      unmergeList,
+      unmergeLastCollapsed,
+      unmergeDisabled,
+      enableFingerprintCompare,
+    } = GroupingStore;
+
+    return {
+      enableFingerprintCompare,
+      unmergeList,
+      unmergeLastCollapsed,
+      unmergeDisabled,
+    };
+  }
+
+  componentWillUnmount() {
+    callIfFunction(this.listener);
```

Comment:
Could you switch to `this.listener?.()`, We should audit `callIfFunction`

Reply:
sure thing  ;-)